addons/weaponselect/functions/fnc_selectPistolFast.sqf

```diff
@@ -0,0 +1,25 @@
+// by commy2
+#include "script_component.hpp"
+
+params ["_unit", "_weapon"];
+
+local _animIndex = [//@todo add anims here
+    "amovpercmstpsraswrfldnon",
+    "amovpercmrunsraswrfldf", "amovpercmrunsraswrfldb", "amovpercmrunsraswrfldl", "amovpercmrunsraswrfldr", "amovpercmrunsraswrfldbl", "amovpercmrunsraswrfldbr", "amovpercmrunsraswrfldfl", "amovpercmrunsraswrfldfr",
+    "amovpknlmrunsraswrfldf", "amovpknlmrunsraswrfldb", "amovpknlmrunsraswrfldl", "amovpknlmrunsraswrfldr", "amovpknlmrunsraswrfldbl", "amovpknlmrunsraswrfldbr", "amovpknlmrunsraswrfldfl", "amovpknlmrunsraswrfldfr"
+] find animationState _unit;
```

Comment:
If not, we should definitely put them in a define/variable for this package. Don't repeat yourself.